server/utils/file/csvParser.js

```diff
@@ -83,6 +83,7 @@ class CSVParser {
 
     if (this._headers === null) {
       this._headers = data
+      console.log('headers', this._headers, this.headers)
```

Comment:
we don't want to use console.log anymore. please use Logger

Reply:
I'll remove it, even csvParser (used in taxonomyImportJob still) should be replaced by csvReader

Reply:
yes, let's postpone it though